validator/src/main.rs

```diff
@@ -1478,6 +1480,32 @@ pub fn main() {
                 .takes_value(true)
                 .help("Add a hard fork at this slot"),
         )
+        .arg(
+            Arg::with_name("broadcast_stage_type")
+                .long("broadcast-stage-type")
+                .value_name("TYPE")
+                .takes_value(true)
+                .possible_values(&[
+                    "standard",
+                    "fail_entry_verification",
+                    "broadcast_fake_shreds",
+                    "broadcast_duplicates"])
+                .help(
+                    "Broadcast stage type, defaults to standard. Used to simulate malicious validator
+                    configurations."
+                ),
+        )
+        .arg(
+            Arg::with_name("duplicate_stake_partition")
+                .long("duplicate-stake-partition")
+                .value_name("STAKE")
+                .takes_value(true)
+                .requires("broadcast_stage_type")
+                .default_value("50")
+                .help("Specifies the percentage of the stake to send duplicate shreds to when using
+                    the broadcast_duplicates stage type."
+                ),
+        )
```

Comment:
At minimum we want to hide these options from the --help.  We might even want to go further and not even land the fully plumbing for this until @carllin's slot dumping work completes and rolls out to mainnet

Reply:
Fair point, definitely don't want to land this until slot dumping has been rolled out. I created this pull request so that we could test @carllin's prs on the gce cluster. I'll also remove the help info.

Reply:
Got it, thank!